apps/modernization-ui/src/pages/patient/profile/ethnicity/Ethnicity.tsx

```diff
@@ -55,8 +63,13 @@ export const Ethnicity = ({ patient }: Props) => {
     const { profile } = usePatientProfile(id);
 
     const handleComplete = (result: FindPatientProfileQuery) => {
-        setData(asView(result.findPatientProfile?.ethnicity));
-        setEntry(asEntry(result.findPatientProfile?.ethnicity));
+        const current = result.findPatientProfile?.ethnicity;
+        if (!current) {
+            setEntry(newEntry(current));
+        } else {
+            setData(asView(current));
+            setEntry(asEntry(current));
+        }
     };
 
```

Comment:
General Patient Information, Mortality, and Sex at Birth are returned as valid search objects with an "asOf" even if there is not data.